geopandas/io/sql.py

```diff
@@ -92,8 +96,11 @@ def load_geom_text(x):
             srid = shapely.get_srid(geoms.iat[0])
             # if no defined SRID in geodatabase, returns SRID of 0
             if srid != 0:
-                crs = "epsg:{}".format(srid)
-
+                if spatial_ref_sys_df is not None:
+                    entry = spatial_ref_sys_df.loc[spatial_ref_sys_df["srid"] == srid]
```

Comment:
Thanks @m-richards ,

I've added an additional check that the `srid` is in the `spatial_ref_sys` table, otherwise falling back to epsg and added a user warning when it does this. I've written a test to check this using `mock.patch`